adserver/tasks.py

```diff
@@ -934,8 +932,9 @@ def disable_inactive_publishers(days=60, draft_only=False, dry_run=False):
     threshold = get_ad_day() - datetime.timedelta(days=days)
     site = get_current_site(request=None)
 
+    # Don't disable publishers who we recently updated to paid
     for publisher in Publisher.objects.filter(
-        allow_paid_campaigns=True, created__lt=threshold
+        allow_paid_campaigns=True, created__lt=threshold, modified__lt=threshold
```

Comment:
Hrm, I realize we're actually modifying every publisher with the CTR average each night... so this probably won't work.

I'll leave this PR open in case we have a better idea for how to fix this issue.

Reply:
One possibility is to change the query that updates the publisher nightly to only update paid publishers and/or to use `update()` instead of `save()` to not update the modified time.